marshmallow_jsonapi/fields.py

```diff
@@ -22,6 +25,12 @@ class BaseRelationship(Field):
     pass
 
 
+def stringify(value):
```

Comment:
Ok. Why do we actually need this? I realized that while the IDs in the main data object are integers, the IDs in the linkages are strings.